### DISCUSSION TITLE
Why is the App Version in F-droid stuck at 0.32, yet the gplay-store & github provide 0.42?

### DISCUSSION THREAD

**Discussion by 1reverseengineer** [moneytoo/Player#143] [Q&A] [answered]
Fdroid shows the newest version is 0.32, yet when you look at google play or this repo, the latest version is 0.42.
Why is there such a big version discrepancy compared to all the other stores?

I mean usually google play takes hours or even days to review/permit updates, but I can't imagine why f-droid is lacking behind.

To the developer only: Thanks for creating this wonderful app, it beats everything else due to it's ability to play almost any format (especially those high quality/bitrate dolby etc... ones), has a UI that doesn't overwhelm you and looks decent enough. And it's free. Thanks!

**Comment by moneytoo** [comment, marked as answer]
Because of the ExoPlayer's FFmpeg extension, F-Droid requires manual update. And because of all the delays, I do that about every month. Now it's due but I'm waiting for an ExoPlayer update (to fix https://github.com/google/ExoPlayer/issues/9061 and https://github.com/google/ExoPlayer/issues/9086).

**Reply by 1reverseengineer** [reply]
Is there any way I can help you with just (video) player? Anything, testing, reports, code?

**Reply by moneytoo** [reply]
You can translate the few strings to a new language on Weblate, but a lot of languages are already covered. Bug reports are always welcome. The code is a mess and I know that - it's just my place to have fun and learn, not really to collaborate. But there is a lot of requests directly in ExoPlayer so anything you help with there will impact a lot of players.

**Comment by moneytoo** [comment]
Version [0.45 was merged](https://gitlab.com/fdroid/fdroiddata/-/commit/3ec7ce26fc9fc606a841d59b26296338eb3fe577) to Fdroid so it should become available soon (once it's built, signed and released  by Fdroid in 2 to 5 days).
